lib/floe/workflow/states/task.rb

```diff
@@ -65,7 +64,6 @@ def retry!(retrier)
 
         def process_output!(output, results)
           return output if results.nil?
-          return if output_path.nil?
```

Comment:
Please verify that we want no output if output_path is nil.
Is it a configuration error, or a known use case.
I probably broke this behavior.

the other `input_path`/`output_path` implementation seemed to blow up in this case.